test/dummy/app/components/dummy/mailer/footer_component.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Dummy::Mailer::FooterComponent < ApplicationComponent
+  def initialize(site:, logo: false, menu: nil)
```

Comment:
bez `logo`  se to nikde nepouziva, muzes vyhodit

Reply:
Pridaval jsem to tam na zaklade poznamky "Optional footer logo" ve Figme 
![image](https://github.com/sinfin/folio/assets/83029101/c1340f63-270d-4d45-bfb7-b82ca92e8670)

Reply:
optional bude tim, ze se kdyztak ve vygenerovane sablone smaze, neni potreba jako argument